stack2cabal.cabal

```diff
@@ -43,7 +43,7 @@ common deps
     , filepattern           ^>=0.1.2
     , fuzzy-dates           ^>=0.1.1.2
     , hourglass             ^>=0.2.12
-    , hpack                 ==0.34.2
+    , hpack                 <=0.34.2
```

Comment:
It seems this needs to be 0.34.3:

```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for stack2cabal-1.0.12:
    hpack-0.34.3 from stack configuration does not match <=0.34.2  (latest matching version
                 is 0.34.2)
needed since stack2cabal is a build target.

Some different approaches to resolving this:

  * Set 'allow-newer: true'
    in /home/hasufell/.stack/config.yaml to ignore all version constraints and build anyway.

  * Recommended action: try adding the following to your extra-deps
    in /home/hasufell/git/stack2cabal/stack.yaml:

- hpack-0.34.2@sha256:9ada6a470d7f5d8dcd3edc08b0ee2b5d254909ded24f6c6f1afcd9840248a253,4341

Plan construction failed.
```

Reply:
`== 0.34.3`?